src/server/service/kv.rs

```diff
@@ -811,6 +812,13 @@ impl<E: Engine, L: LockManager, F: KvFormat> Tikv for Service<E, L, F> {
         let res = async move {
             let mut stream = stream.map_err(Error::from);
             while let Some(mut batch_msg) = stream.try_next().await? {
+                let now = SystemTime::now()
+                    .duration_since(UNIX_EPOCH)
+                    .unwrap()
+                    .as_nanos() as u64;
+                let elapsed = nanos_to_secs(now.saturating_sub(batch_msg.last_observed_time));
+                RAFT_MESSAGE_DURATION.send.observe(elapsed);
```

Comment:
Can you add sender store id to the metrics? Because here the tikv is a receiver and it observes the send time for sender tikv nodes.

Reply:
I think this may be unnecessary, because the network issues are usually bidirectional, and the sender can also detect the abnormal send duration.

Reply:
Em, the point is that `send` is misleading. How about renaming it to receive_delay instead?

Reply:
Nice name!